Sources/Commands/Completions+zsh.swift

```diff
@@ -0,0 +1,127 @@
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import Basic
+import Utility
+
+/// Template for ZSH completion script.
+///
+/// - Parameter stream: output stream to write the script to.
+func zsh_template(on stream: OutputByteStream) {
+    stream <<< "#compdef swift\n"
+    stream <<< "local context state state_descr line\n"
+    stream <<< "typeset -A opt_args\n"
+    stream <<< "\n"
+    stream <<< "_swift() {\n"
+    stream <<< "    declare -a shared_options\n"
+    stream <<< "    shared_options=(\n"
```

Comment:
The variable is a left-over from refactoring I guess. I don't think it should even be there anymore.